File: trunk/Client/src/ru/ipo/dces/clientservercommunication/AdjustClientPluginRequest.java
package ru.ipo.dces.clientservercommunication;

/**
 * Created by IntelliJ IDEA.
 * User: Илья
 * Date: 18.12.2008
 * Time: 23:30:51
 */
public class AdjustClientPluginRequest implements Request {

  public String sessionID;
  public String pluginAlias;
  public byte[] pluginData;
  public String description;

}

File: Server/tests/data/AdjustClientPluginRequest.php
<?php
class AdjustClientPluginRequest{
    public $sessionID;
    public $pluginAlias;
    public $pluginData;
    public $description;

    function __construct(){
        $this->sessionID = null;
        $this->pluginAlias = 'Test client plugin';
        $this->pluginData = 'jar file content';
        $this->description = 'Test client plugin description';
    }
};
?>
